e/envy/envy_ubi8.3.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package        : Envy
+# Version        : 1.7.1 - latest
+# Source repo    : https://github.com/gobuffalo/envy
+# Tested on      : Ubi 8.3 + Go latest version (v1.7.1)
+# Script License : Apache 2.0
+# Maintainer     : Eshant Gupta <eshant.gupta@ibm.com>; Priya Seth <sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# Usage: ./envy_ubi_8.3.sh
+#                  OR
+# 	 ./envy_ubi_8.3.sh v1.7.1
+# ----------------------------------------------------------------------------
+
+source ~/.profile
+set_env=false
+
+# Get sudo access
+[ "$UID" -eq 0 ] || exec sudo bash "$0" "$@"
+
+function usage() {
+	echo "$1 accepts 0 or maximum 1 argument"
+	exit 1
+}
+
+if [ "$#" -gt 1	]
+then
+	usage "$(basename $0)"
+else
+	[ -n "$1" ] && export TAGS="$1" || export TAGS=""
+fi
+
+# check installed Golang
+[ -n $(which go) ] && [[ $(go version) > 1.6 ]] && set_env=true
```

Comment:
Isn't the latest version 1.17 ?

Reply:
typo fixed, it has to be 1.16. if version is not found or found to be earlier release, then script download the latest go version.